Resources/Prototypes/_ShitChap/Entities/Weapons/Melee/Nullrod.yml

```
@@ -0,0 +1,466 @@
+    soundGunshot: /Audio/Effects/radpulse7.ogg
+    fireRate: 0.5
+    useKey: false
+  - type: GunRequiresWield
+  - type: MeleeWeapon
+    attackRate: 1.0
+    damage:
+      types:
+        Slash: 15
+        Holy: 20
+    soundHit:
+        path: /Audio/Weapons/bladeslice.ogg
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)